metrics-operator/converter/sli_converter.go

```diff
@@ -0,0 +1,78 @@
+package converter
+
+import (
+	"fmt"
+
+	metricsapi "github.com/keptn/lifecycle-toolkit/metrics-operator/api/v1alpha3"
+	"gopkg.in/yaml.v2"
```

Comment:
I'd use [sigs.k8s.io/yaml](https://github.com/kubernetes-sigs/yaml) instead. Behind the scenes, they also use the very same library that you suggested. However, I'd prefer to depend on the "official" K8s wrapper to ensure the manifest we generate are using the exact same library used by the controller runtime